Update dependencies

phoenix6936 opened the pull request:
Add dependabot for workflow and gradle dependencies
Update dependencies gradle
This PR is appropriate because dependabot informed gradle dependencies and workflows dependencies
@2dust

Commit:
Create dependabot.yml

Add dependabot for workflow and gradle dependencies

Commit:
Update libs.versions.toml

Update dependencies gradle

dependabot[bot] committed:
Bump io.reactivex.rxjava3:rxjava from 3.1.9 to 3.1.10 in /V2rayNG

Bumps [io.reactivex.rxjava3:rxjava](https://github.com/ReactiveX/RxJava) from 3.1.9 to 3.1.10.
- [Release notes](https://github.com/ReactiveX/RxJava/releases)
- [Commits](https://github.com/ReactiveX/RxJava/compare/v3.1.9...v3.1.10)

---
updated-dependencies:
- dependency-name: io.reactivex.rxjava3:rxjava
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Commit:
Merge pull request #4 from phoenix6936/dependabot/gradle/V2rayNG/io.reactivex.rxjava3-rxjava-3.1.10

Bump io.reactivex.rxjava3:rxjava from 3.1.9 to 3.1.10 in /V2rayNG

Commit:
Merge branch '2dust:master' into master

ghost commented:
It makes your job easier to update dependencies. It is a very good thing if you decide to merge
@2dust

Commit:
Merge branch '2dust:master' into master

dependabot[bot] committed:
Bump org.jetbrains.kotlin.android from 2.0.21 to 2.1.0 in /V2rayNG

Bumps [org.jetbrains.kotlin.android](https://github.com/JetBrains/kotlin) from 2.0.21 to 2.1.0.
- [Release notes](https://github.com/JetBrains/kotlin/releases)
- [Changelog](https://github.com/JetBrains/kotlin/blob/master/ChangeLog.md)
- [Commits](https://github.com/JetBrains/kotlin/commits)

---
updated-dependencies:
- dependency-name: org.jetbrains.kotlin.android
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

Commit:
Merge pull request #5 from phoenix6936/dependabot/gradle/V2rayNG/org.jetbrains.kotlin.android-2.1.0

Bump org.jetbrains.kotlin.android from 2.0.21 to 2.1.0 in /V2rayNG

Commit:
Update README.md

https://github.com/JetBrains/kotlin/releases/tag/v2.1.0
Update kotlin version to 2.1.0

Commit:
Update kotlin version to 2.1.0

Update kotlin version to 2.1.0

Commit:
Update kotlin version to 2.1.0

Update kotlin version to 2.1.0

Commit:
Update kotlin version to 2.1.0

Update kotlin version to 2.1.0

Commit:
Update kotlin version to 2.1.0

Update kotlin version to 2.1.0

2dust commented:
You don’t need to always use the latest version, because each update needs to be tested and verified.

ghost commented:
> You don’t need to always use the latest version, because each update needs to be tested and verified.

See how dependabot works, it offers the new version. Do you want to like it or not? Anyway, it was a good thing, you can do whatever you like.
